recipe/meta.yaml

```diff
@@ -27,13 +29,14 @@ requirements:
     # optional deps [socks]
     - pysocks >=1.5.6,<2.0,!=1.5.7
     # optional deps [brotli]
-    - brotlipy >=0.6.0
+    - brotli >=1.0.9
```

Comment:
This should have stayed with `brotlipy`. `brotli` is a C library. Opened #2 about this.